capstone-app/src/views/LandingPage/LandingPage.js

```diff
@@ -16,7 +16,7 @@ import styles from "assets/jss/material-kit-react/views/landingPage.js";
 // Sections for this page
 import ProductSection from "./Sections/ProductSection.js";
 import SectionModal from "./Sections/SectionModal.js";
-import SectionNavbar from "./Sections/SectionNavbarLoggedin.js";
+import SectionNavbarSignup from "./Sections/SectionNavbarSignup.js";
```

Comment:
SectionNavbarSignup is not the name of the component in the file. Use SectionNavbar from SectionNavbarLoggedin instead, as that is the Navbar with the authentication built into it.

Reply:
Done in https://github.com/googleinterns/step160-2020/pull/12/commits/0cc4735f5c74d545d8c9b3c757a2a938d5df88a9